src/components/ModalWindow/ModalWindow.jsx

```diff
@@ -1,8 +1,17 @@
 import React from 'react';
-import {PersonModalWindow} from './PersonModalWindow';
 import {useNavigate} from 'react-router-dom';
+import PersonModalWindow from '../ModalWindow/PersonModalWindow';
+import PlanetModalWindow from '../ModalWindow/PlanetModalWindow';
 
-export const ModalWindow = personUrlId => {
+export const ModalWindow = ({personUrlId, planetUrlId}) => {
   const navigate = useNavigate();
-  return <div>{personUrlId && <PersonModalWindow onClose={() => navigate('/people')} />}</div>;
+  return (
+    <div>
+      <div>
+        {personUrlId && <PersonModalWindow onClose={() => navigate('/people')} />}
+
+        {planetUrlId && <PlanetModalWindow onClose={() => navigate('/planets')} />}
```

Comment:
It is a very bad practice. ModalWindow should be an independent component. It shouldn't know anything about Person or Planets. It should not handle navigation